docs/features/user_experience.md

```diff
@@ -4,10 +4,87 @@ The end user leverages the web interface as the primary method to engage with th
+
+## Work Only
+
+Work Only mode provides the natural language conversation with your data that you are familiar with from previous versions of Information Assistant.
+
+## Work + Web
+
+The Work + Web mode experience starts similar to Work Only mode. It provides new options in the response enabling the user to __Searh Web__ to see if there is newer information available or __Compare with Web__ to compare the response from their data with information available on the web. 
+
+![Chat Link](/docs/images/work-plus-web-ui.png)
+
+Selecting either of these options changes the user experience as follows:
+
+### Search Web
+
+Selecting __Search Web__ from a Work Only response asks the same question on the web. The UI color scheme changes to reflect the user is searching a different source.
+
+![Chat Link](/docs/images/work-plus-web-search-web.png)
+
+Options in the __Search Web__ response include Search Work and Compare with Work.
+
+### Compare with Web
+
+Selecting __Compare with Web__ from a Work Only response asks the same question on the web then compares your data response with the response from the web. The UI color scheme changes to reflect the user is comparing Work with Web
+
+![Chat Link](/docs/images/work-plus-web-compare-with-web.png)
+
+The Compare with Web response includes both work and web citations.
+
+## Generative
+
+Generative mode enables user interaction with the LLM without grounding the response in their data.
+
+![Chat Link](/docs/images/generative-ungrounded-ui.png)
+
+On the top right are two new preview features called Math Assistant and Tabular Data Assistant. The Math Assistant and Tabular Data assistant are being released in preview mode as we continue to evaluate and mitigate the potential risks associated with autonomous reasoning Agents, such as misuse of external tools, lack of transparency, biased outputs, privacy concerns, and remote code execution vulnerabilities. With future release we plan work to enhance the safety and robustness of these autonomous reasoning capabilities.
+
+## Math Assistant
+
+The Math Assistant combines natural language understanding with robust mathematical reasoning, enabling users to express mathematical queries in plain language and receive step-by-step solutions and insights.
+
+![Chat Link](/docs/images/math-assistant-ui.png)
+
+Users can select an example query or enter their own math question. There are three options below the question promt where the user can ask for clues, ask the assistant to show them how to solve the math problem, or simply provide an answer.
+
+### Give me clues
+
+Give me clues will assist the user without providing the answer or showing them how to solve the problem.
+
+![Chat Link](/docs/images/math-assistant-give-me-clues.png)
```

Comment:
typically we highlight in the image with a red box

Reply:
created new screen captures then used paint to add a red box around the area of interest